portfolio/src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps.servlets;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+ 
+// Servlet that returns the status of a user.
+@WebServlet("/user")
+public class UserServlet extends HttpServlet {
+ 
+@Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    String json;
+    if (userService.isUserLoggedIn()) {
+      json = convertToJsonUsingGson("You are in!");
+    } else {
+      json = convertToJsonUsingGson("Please log in!");
+    }
+    // Send the JSON as the response.
```

Comment:
nit: As a style preference, can you please include an extra space between the end of the if-else bracket and the following lines?

```suggestion
    }
    
    // Send the JSON as the response.
```

Reply:
Ack!